crates/rerun/src/run.rs

```diff
@@ -547,33 +583,62 @@ fn native_viewer_connect_to_ws_url(
     Ok(())
 }
 
-fn load_file_to_channel(path: &std::path::Path) -> anyhow::Result<Receiver<LogMsg>> {
+#[allow(clippy::needless_pass_by_value)] // false positive on some feature flags
+fn load_file_to_channel_at(
+    recording_id: re_log_types::RecordingId,
+    path: &std::path::Path,
+    tx: re_smart_channel::Sender<LogMsg>,
+) -> Result<(), anyhow::Error> {
+    puffin::profile_function!(path.to_string_lossy());
+    re_log::info!("Loading {path:?}…");
+
+    let extension = path
+        .extension()
+        .unwrap_or_default()
+        .to_ascii_lowercase()
+        .to_string_lossy()
+        .to_string();
+
+    if extension == "rrd" {
+        load_rrd_file_to_channel(path.to_owned(), tx)
+    } else {
+        #[cfg(feature = "sdk")]
+        {
+            let log_msg = re_sdk::MsgSender::from_file_path(path)?.into_log_msg(recording_id)?;
+            tx.send(log_msg).ok();
```

Comment:
This needs a comment explaining why we're fine with silently ignoring that the channel is closed here: is this because it can never happen? or because we don't care if it does? when can it happen?